CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## Unreleased
+
+Special thanks to external contributors on this release:
```

Comment:
also do you want this sentence here with the empty section

Reply:
since it's in unreleased section, I would say it's fine but I am willing to remove. don't really have a preference